cmd/codegen/generator/typescript/templates/src/testdata/type_test_enum_with_directive_want.ts

```diff
@@ -0,0 +1,49 @@
+
+/**
+ * Compression algorithm to use for image layers.
+ */
+export enum ImageLayerCompression {
+  EstarGz = "EStarGZ",
+  Estargz = "EStarGZ",
+  Gzip = "Gzip",
+  Uncompressed = "Uncompressed",
+  Zstd = "Zstd",
+}
```

Comment:
I think this is supported in typescript?

```suggestion
export enum ImageLayerCompression {
  EstarGz = "EStarGZ",
  Estargz = EstarGz,
  Gzip = "Gzip",
  Uncompressed = "Uncompressed",
  Zstd = "Zstd",
}
```

That way, `ImageLayerCompression.EstarGz == ImageLayerCompression.Estargz` which is a pretty handy property (we do this in go).